centralserver/i18n/management/commands/update_language_packs.py

```diff
@@ -1,858 +0,0 @@
-"""
```

Comment:
Ah, yup. If we still want to keep building 0.15 lang packs (which I'm assuming we do), then we need to retain these.